lock-api/src/main/java/com/palantir/lock/client/metrics/TimeLockFeedbackBackgroundTask.java

```diff
@@ -69,16 +77,20 @@ public static TimeLockFeedbackBackgroundTask create(TaggedMetricRegistry taggedM
     public void scheduleWithFixedDelay() {
         executor.scheduleWithFixedDelay(() -> {
             try {
+                ConjureTimeLockClientFeedback feedbackReport = ConjureTimeLockClientFeedback
+                        .builder()
+                        .startTransaction(getEndpointStatsForStartTxn())
+                        .leaderTime(getEndpointStatsForLeaderTime())
+                        .atlasVersion(versionSupplier.get())
+                        .nodeId(nodeId)
+                        .serviceName(serviceName)
+                        .build();
+                timeLockClientFeedbackServices
+                        .get()
```

Comment:
Currently this creates the clients every time, which is not ideal: maybe the BroadcastDialogueServiceProvider could return a more helpful type?